_drafts/dropin-starter-template.md

```diff
@@ -0,0 +1,63 @@
+# Using the Adobe Commerce Drop-in Template Repository
+
+This guide helps developers and partner agencies quickly create their own drop-in applications by leveraging Adobe Commerce's Drop-in Template repository.
```

Comment:
```suggestion
This topic describes how to use the `drop-template` repository to create drop-in components for Adobe Commerce Storefronts.
```

Reply:
Same question here regarding the term "drop-in components"